client/my-sites/hosting/sftp-card/index.js

```diff
@@ -239,8 +241,21 @@ export const SftpCard = ( {
 				"Access and edit your website's files directly by creating SFTP credentials and using an SFTP client."
 		  );
 
+	const [ isTwoStepEnabled, setTwoStepEnabled ] = useState(
```

Comment:
Would it make sense to flip up the condition and call the variable `isReauthRequired`? If we say "enabled", we might think it's about checking if the 2FA feature is enabled, but here we are checking if 2FA is required for a given session.

Reply:
yeah that make sense.